github/provider.go

```diff
@@ -239,13 +249,52 @@ func providerConfigure(p *schema.Provider) schema.ConfigureFunc {
 			}
 
 			token = appToken
-		}
 
-		writeDelay := d.Get("write_delay_ms").(int)
-		if writeDelay <= 0 {
-			return nil, fmt.Errorf("write_delay_ms must be greater than 0ms")
+			if owner == "" {
+				ctx := context.Background()
+
+				// Authenticate using the App Installation `token`
+				ts := oauth2.StaticTokenSource(
+					&oauth2.Token{AccessToken: token},
+				)
+
+				client := oauth2.NewClient(ctx, ts)
+				c := Config{
+					Token:      token,
+					BaseURL:    baseURL,
+					Insecure:   insecure,
+					WriteDelay: time.Duration(writeDelay) * time.Millisecond,
+				}
+
+				v3client, err := c.NewRESTClient(RateLimitedHTTPClient(client, c.WriteDelay))
+
+				if err != nil {
+					return nil, err
+				}
+
+				var o Owner
+				o.v3client = v3client
```

Comment:
Can this be simplified to calling `v3client` directly?  It looks like we allocate an `Owner` here but do not use it meaningfully.